pyproject.toml

```
@@ -41,7 +41,7 @@ dependencies = [
   "napari[all]",
   "qtpy",
   "tox",
-  "TopoStats @ git+https://github.com/AFM-SPM/TopoStats.git@main",
+  "topostats",
```

If there is a minimum version then the need to raise errors if old versions are being used can be eliminated by setting the minimum version here.

```suggestion
  "topostats>=2.3.2",
```